jsonrpc/dispatcher.go

```diff
@@ -129,6 +130,7 @@ func (d *Dispatcher) handleSubscribe(req Request, conn wsConn) (string, Error) {
 
 	subscribeMethod, ok := params[0].(string)
 	if !ok {
+		//nolint:forcetypeassert
```

Comment:
Why do we not force only for string type assertions?

Reply:
🤦 
We actually already do type assertion in the line before 👍 

Fixed 👍 
87fd58f